_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+---
+title: The Western Grid, Explained
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2019-02-24 08:00:00
+categories:
+  - Featured
+tags:
+  - plss
+  - history
+---
+
+<figure class="caption caption--right">
+  <a href="{{ "/images/SaltLakeHistoric.jpg" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/SaltLakeHistoric_sm.jpg" | prepend: site.baseurl }}" alt="1891 Salt Lake City Bird's Eye View">
+  </a>
+  <figcaption class="caption__text">Fig 1: 1891 Salt Lake City Bird's Eye View</figcaption>
+</figure>
+
+Have you ever wondered why the patterns of development in the Western United States are so orthogonal (i.e., "right-angled")? Pan around a map and you'll see the confines of the grid just about everywhere. Grid-like development is nothing new - it can be traced back to early developments in the Middle East. But, why is it so prevalent in the West?
+
+It turns out this gridiron pattern is connected to a 17th-century mathematician, the Treaty of Paris, and Thomas Jefferson. Yup, there's definitely a connection here, which also helps explain the West's notoriously wide streets and big city blocks, as well as our nation’s entangled dependence on feet, yards, and miles (i.e., the United States customary units). Read on and we will explore the Western Grid more.
+
+## The Survey Chain
+{: .text-left}
+
+<figure class="caption caption--right">
+  <img src="{{ "/images/GunterChain.png" | prepend: site.baseurl }}" alt="Gunter Chain" />
+  <figcaption class="caption__text">Fig 2: Gunter Chain</figcaption>
```

Comment:
K, this should be footnote 2 (and the new/updated image that I added). Footnote should be: 

“Gunter Chain,” (image) John Johnson (owner), Smithsonian National Museum of American History, accessed March 01, 2019. http://americanhistory.si.edu/collections/search/object/nmah_761634.

Reply:
@atribolet the image seems to be missing in the preview.  maybe it's the space in the image file name?